src/main/java/net/imglib2/realtransform/AbstractRealTransformSequence.java

```diff
@@ -102,6 +103,12 @@ public int numTargetDimensions()
 		return nTarget;
 	}
 
+	@Override
+	public boolean isIdentity()
+	{
+		return transforms.isEmpty();
```

Comment:
This is too restrictive, isn't it? If this sequence actually contains an identity transform, or if it contains only an `InvertibleRealTransform` and its inverse, I would expect `isIdentity` to return `true`.

Maybe we could make use of `RealViewsSimplifyUtils.isIdentity()` here, as in the other transforms?

Or, if this is to be kept, I vote for adding a comment to the Javadoc of `RealTransform#isIdentity`, stating that if `false` it *could* still be identity.

Reply:
Yea, thanks for mentioning this. 

I'm also inclined to using `RealViewsSimplifyUtils.isIdentity()`